backend/Services/SightingService.cs

```diff
@@ -51,8 +54,21 @@ public Sighting CreateSighting(CreateSightingRequest request)
                 Longitude = request.Longitude,
                 ConfirmationStatus = ConfirmationStatus.Pending,
             };
-
             return _sightings.CreateSighting(newSighting);
         }
+
+        public Sighting ConfirmOrRejectSighting(ConfirmOrRejectRequest confirmOrRejectSighting, int sightingId)
+        {
+            if (confirmOrRejectSighting.NewConfirmationStatus == ConfirmationStatus.Approved)
+            {
+                return _sightings.ConfirmRequest(sightingId);
+            }
+            else if (confirmOrRejectSighting.NewConfirmationStatus == ConfirmationStatus.Rejected)
+            {
+                return _sightings.RejectRequest(sightingId);
+            }
+            throw new ArgumentOutOfRangeException("The confirmation request was not to approve or reject the sighting." +
```

Comment:
In concatenating these strings, you've lost a space:
```suggestion
            throw new ArgumentOutOfRangeException("The confirmation request was not to approve or reject the sighting. " +
```